x-pack/plugins/index_lifecycle_management/public/application/sections/policy_list/components/policy_table.tsx

```diff
@@ -54,8 +77,36 @@ export const PolicyTable: React.FunctionComponent<Props> = ({ policies }) => {
   const {
     services: { getUrlForApp },
   } = useKibana();
-
+  const [managedPoliciesVisible, setManagedPoliciesVisible] = useState<boolean>(false);
   const { setListAction } = usePolicyListContext();
+  const searchOptions = useMemo(
+    () => ({
+      box: { incremental: true, 'data-test-subj': 'ilmSearchBar' },
+      toolsRight: (
+        <EuiFlexItem grow={false}>
+          <EuiSwitch
```

Comment:
As discussed, would be nice to remember the setting of this switch, so that it doesn't revert to 'off' after viewing or editing a managed policy.